webpack.config.js

```diff
@@ -17,20 +19,50 @@ module.exports = (env) => {
           use: ["babel-loader"],
         },
         {
-          test: /\.(css|sass)$/i,
-          use: ["style-loader", "css-loader", "sass-loader"],
+          test: /\.css$/i,
+          use: [MiniCssExtractPlugin.loader, "css-loader"],
         },
         {
-          test: /\.(png|svg|jpg|jpeg|gif)$/i,
+          test: /\.sass$/i,
+          use: ["sass-loader"],
+        },
```

Comment:
Do you want to use SASS in this project? If no, let's just remove the loader config for `.sass` files. If yes, then I believe this config is invalid, as you need `css-loader` and `MiniCssExtractPlugin.loader` for `.sass` files too.

Reply:
I removed